config/cms.php

```diff
@@ -93,6 +93,19 @@
 
     'backendSkin' => 'Backend\Skins\Standard',
 
+    /*
+    |--------------------------------------------------------------------------
+    | Determines if logging in backend should run UpdateManager
+    |--------------------------------------------------------------------------
+    |
+    | If value is true it runs UpdateMananger after login in backend.
+    | It's recommended to set this value to 'false' in production enviroments
+    | because it cleares cache on every login in backend.
```

Comment:
@Samuell1 Grammar fix.
```suggestion
    | because it clears the cache every time a user logs in to the backend.
```